### PR TITLE
adding examples in Golang

### PR DESCRIPTION
**Pull request by DucarrougeR**

### CODE REVIEW

File: examples/go/get_all_dashboards.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/looker-open-source/sdk-codegen/go/rtl"
+	v4 "github.com/looker-open-source/sdk-codegen/go/sdk/v4"
```
Comment (kppk) [inline examples/go/get_all_dashboards.go:7]: super nit: seems the alias `v4` is not needed?

**Review by kppk — DISMISSED** [review]
lgtm, thanks! :+1:

File: examples/go/get_all_dashboards.go (hunk 1 above)
Comment (DucarrougeR) [reply examples/go/get_all_dashboards.go:7]: Great, thanks I am updating these

Comment (DucarrougeR) [reply examples/go/get_all_dashboards.go]: Closing this comment here, I've updated all 4 files to remove the alias. Thanks again

**Review by kppk — APPROVED** [review]
lgtm

**Pull request merged**